pkg/devserver/server.go

```diff
@@ -31,25 +33,76 @@ func (s *Server) Start() error {
 	return nil
 }
 
+type PageTemplate struct {
+	Title   string
+	Content string
+}
+
+var pageTemplateContent = `<!DOCTYPE html>
+<html>
+	<head>
+		<title>{{ .Title }}</title>
+		<link rel="stylesheet" href="/assets/css/main.css">
+	</head>
+	<body>
+		<h1>{{ .Title }}</h1>
+		{{ .Content }}
+	</body>
+</html>`
+
+var pageTemplate *template.Template = nil
```

Comment:
```suggestion
var pageTemplate *template.Template
```
`nil` is the zero value for a pointer, so this assignment is unnecessary